src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./footer.css";
+import Github from "../../assets/image/HomePage-Images/Github.png";
+import Twitter from "../../assets/image/HomePage-Images/Twitter sds.png";
+import LinkedIn from "../../assets/image/HomePage-Images/LinkedIn.png";
```

Comment:
you can add the images in public folder to directly use it instead of importing each image as a component

Reply:
I will add images folder in public

Reply:
Thankyou.